server/src/domain/user/user.service.ts

```diff
@@ -120,6 +121,14 @@ export class UserService {
     return this.userCore.getUserProfileImage(user);
   }
 
+  async getUserProfileImageHash(userId: string): Promise<string> {
```

Comment:
This shouldn't be necessary. I believe a weak etag is automatically generated.

Reply:
thanks, i realized that a moment ago too. I will revert the change and just set cache-control to nocache